docs/getting-started/key-terms.md

```diff
@@ -0,0 +1,13 @@
+# Key Terms
+
+## Records
+
+Records are the base of Hubspot, and what we build out from, plainly, our records top line is the transit provider or organization and contains all the activity that has occurred with that organization. Creating records enables us to find information on organization or provider’s relationships. Once created, you can store information such as individual contact information, tickets, documents, meeting notes, last activity and any custom properties we have assigned. Records are continually updated and maintained by all users whether that be updating contact information, merged agencies or activity that would be relevant to other user’s understanding and workflows.
```

Comment:
>...and what we build out from, plainly, our records top line is the...

Edit:...and what we build out from. Our records top line is the...

Reply:
Yeah, @laneymangan, I see what you mean. They are all considered records (contacts, tickets, deals etc.) so you can have your own properties with each. I will need to reword this section all together.